ci-operator/step-registry/baremetal/lab/pre/proxy/baremetal-lab-pre-proxy-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+[ -z "${AUX_HOST}" ] && { echo "AUX_HOST is not filled. Failing."; exit 1; }
+
+if [ x"${DISCONNECTED}" != x"true" ]; then
+  echo 'Skipping proxy configuration'
+  exit
+fi
+
+# ipi-conf-proxy will run only if a specific file is found, see step code
+
+cp "${CLUSTER_PROFILE_DIR}/proxy" "${SHARED_DIR}/proxy_private_url"
```

Comment:
add this to the pre-firewall command when a new template variable (`CLUSTER_WIDE_PROXY[=false  by default]`) is set to 'true'.